python/GafferSceneTest/InstancerTest.py

```diff
@@ -2630,6 +2630,50 @@ def testRootPerVertexWithEmptyPoints( self ) :
 		self.assertEqual( instancer["out"].childNames( "/points/instances" ), IECore.InternedStringVectorData() )
 		self.assertSceneValid( instancer["out"] )
 
+	def testPurpose( self ):
+
+		points = IECoreScene.PointsPrimitive(
+					IECore.V3fVectorData(
+						[ imath.V3f( i, 0, 0 ) for i in range( 100 ) ]
+					)
+				)
+
+		points["intVar"] = IECoreScene.PrimitiveVariable( IECoreScene.PrimitiveVariable.Interpolation.Vertex, IECore.IntVectorData(
+						[ i for i in range( 100 ) ]
+					) )
```

Comment:
Trivial I know, but could you fix the indentation so things close at the same indentation level as the line they opened on please?